src/Mvc/Mvc.Core/src/Infrastructure/DefaultStatusCodeAttribute.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.AspNetCore.Mvc.Infrastructure;
 /// <remarks>
 /// This attribute is informational only and does not have any runtime effects.
 /// </remarks>
-[AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = true)]
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
```

Comment:
We should FYI this thru API review. Maybe the comment for this type should be updated to say what it means to apply it to a Method (more as a note to future selves).

Reply:
Yep! Sounds sensible.